src/components/useraccount/UserProfile.vue

```diff
@@ -1,103 +1,28 @@
 <template>
-  <div v-if="me"
-       class="my-account-content">
+   <div :class="{ 'col-sm-9': showTab.showPersonalDetails,
+                  'col-sm-12': showTab.showMyOrders }">
```

Comment:
I feel this means that for every new tab, we need to add it here to decide the class. That's a strong dependency.
Think of another solution that doesn't require to modify this for every tab added.
For example, we know looking at the demo that all of the tabs are `col-sm-12` except for `showPersonalDetails`. That could be a way of avoiding to put all the others in the list.